service/frontend/adminHandler.go

```diff
@@ -623,20 +626,20 @@ func (adh *AdminHandler) validateGetWorkflowExecutionRawHistoryV2Request(
 		return &gen.BadRequestError{Message: "Invalid PageSize."}
 	}
 
-	if request.StartEventId == nil &&
-		request.StartEventVersion == nil &&
-		request.EndEventId == nil &&
-		request.EndEventVersion == nil {
+	if request.GetStartEventId() == common.EmptyEventID &&
```

Comment:
This is the only one problematic subject to discuss. Before, `EventId` and `Version` could be `nil`. I changed usage to `GetEventId()` and `GetVersion()` everywhere. These funcs will never return `nil` but can return `int64` default value (which is `0`). I checked everywhere in code and replaced all `nil` assignments to `common.EmptyEventID` and `common.EmptyVersion` therefore `Get*` funcs should never return `0`. But just in case, if I missed something, or new code will be added later, I would change values of these consts to `0` from `-23` and `-24`. `EventId` starts with `1` and `0` seems to be valid "empty" value. What is the start value for `Version`?

Reply:
Is there a way in proto to define custom default value for a field?  Maybe we can influence code gen to provide our own default value for EventId and Version fields.  That would be ideal.  I'm a little worried about using EmptyVersion as 0 as it could be a valid value.  
I wouldn't block this PR on decision about changing values for EmptyEventID and EmptyVersion.  Please create a separate github issue about changing values for these constants and let's discuss long term approach as part of separate issue.